src/deepsparse/server/main.py

```diff
@@ -273,11 +276,9 @@ def server_app_factory():
 )
 @click.option(
     "--task",
-    type=str,
+    type=click.Choice(SupportedTasks.task_names()),
```

Comment:
Does this need to be optional or something to support using config file without specifying task?

Reply:
Actually nvm i see default is none